src/csharp/Microsoft.Spark/Sql/RelationalGroupedDataset.cs

```diff
@@ -44,6 +44,34 @@ internal RelationalGroupedDataset(JvmObjectReference jvmObject, DataFrame dataFr
         public DataFrame Count() =>
             new DataFrame((JvmObjectReference)_jvmObject.Invoke("count"));
 
+        /// <summary>
+        /// Compute the mean value for each numeric columns for each group.
+        /// </summary>
+        /// <returns>New DataFrame object with mean applied</returns>
```

Comment:
Let's add <param name="colNames">..</param> to each of these.  Can you also add it to Sum() as well.

Reply:
Added